backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/card/BpkCard.kt

```diff
@@ -0,0 +1,74 @@
+package net.skyscanner.backpack.compose.card
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.animation.core.animateDpAsState
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.Role
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkBorderRadius
+import net.skyscanner.backpack.compose.tokens.BpkElevation
+
+enum class BpkCardCorner {
+  Small,
+  Large,
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun BpkCard(
+  modifier: Modifier = Modifier,
+  corner: BpkCardCorner = BpkCardCorner.Small,
+  onClick: (() -> Unit)? = null,
+  interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+  onClickLabel: String? = null,
+  role: Role? = null,
+  content: @Composable () -> Unit,
+) {
+
+  val focused by interactionSource.collectIsFocusedAsState()
+  val pressed by interactionSource.collectIsPressedAsState()
+  val elevated = focused || pressed
```

Comment:
If a card is pressed AND focused it won't look any different to a focused card - is that intended design?
also can multiple cards have a focused state or just one? i believe right now the focused property is set on multiple cards, rather than just a single one

Reply:
We have only one variant of focused card in our design system and I checked with designers what we want it either for focused or pressed state. Focused by definition means something one – do you have any samples when several cards are focused?

Reply:
If you search for the focused property set in the app there's a bunch of cases where it is set to true as a general style, rather than a specifically focused state. This might be against the design system, but probably worth double checking with design if focused is meant as just a temporary state or can be set in general for multiple cards (especially as it is right now)

Reply:
I think it's might be a mistake, but I will check with the design just in case.